lib/src/RedisRemoteSaiInterface.cpp

```diff
@@ -1838,6 +1879,46 @@ sai_status_t RedisRemoteSaiInterface::bulkCreate(
             object_statuses);
 }
 
+sai_status_t RedisRemoteSaiInterface::bulkCreate(
+        _In_ uint32_t object_count,
+        _In_ const sai_my_sid_entry_t* my_sid_entry,
+        _In_ const uint32_t *attr_count,
+        _In_ const sai_attribute_t **attr_list,
+        _In_ sai_bulk_op_error_mode_t mode,
+        _Out_ sai_status_t *object_statuses)
+{
+    SWSS_LOG_ENTER();
+
+    // TODO support mode
+
+    static PerformanceIntervalTimer timer("RedisRemoteSaiInterface::bulkCreate(my_sid_entry)");
```

Comment:
not sure if you need timer for this, timer was originally added for routes, since we have like 65k of them, so we wanted to measure how fast we can push 10k routes, ideally under 1 second
so how many entries this my sid will have ?